src/authz-module/components/RoleCard/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Card, Collapsible, Container, Icon, IconButton,
+} from '@openedx/paragon';
+import { Delete, Person } from '@openedx/paragon/icons';
+import PermissionRow from './PermissionsRow';
+import messages from './messages';
+
+interface CardTitleProps {
+  title: string;
+  userCounter?: number | null;
```

Comment:
I don't see this property being actually passed down, anywhere.  The only instance of RoleCard is in LibrariesUserManager, and it doesn't seem to need `userCounter`.  Will a use for it be added later?

Reply:
Yes, this is a reusable component that displays actions when is in the `LibrariesUserManagement` view, and information when is in the `LibrariesTeamManager` view. 

The use of the prop is relevant for a different PR https://github.com/openedx/frontend-app-admin-console/pull/7